xds/src/test/java/io/grpc/xds/internal/sds/CommonTlsContextTestsUtil.java

```diff
@@ -134,4 +115,27 @@ static CommonTlsContext buildCommonTlsContextWithAdditionalValues(
     }
     return builder.build();
   }
+
+  /**
+   * Helper method to build DownstreamTlsContext for above tests. Called from other classes as well.
```

Comment:
above tests? you mean tests in SdsSslContextProviderTest/ScretVolumnSslContextProviderTest?

Reply:
sorry  - let me clarify